generators/entity/index.js

```diff
@@ -214,8 +213,9 @@ class EntityGenerator extends BaseBlueprintGenerator {
                         context.clientRootFolder = context.microserviceName;
                     }
                 }
+                // TODO 7.0 context.filename = this.destinationPath(context.filename);
```

Comment:
Why not setDestinationPath ?

Reply:
destinationPath is not a setter, it’s a path calculation.
Every yeoman-generator built in method uses destinationRoot as working directory.
destinationPath adds root to the path if the path isn’t absolute.